package.json

```diff
@@ -15,6 +15,7 @@
     "format": "yarn workspace bridge format",
     "test": "yarn workspace bridge test",
     "test-e2e": "yarn workspace bridge test-e2e",
-    "test:fix": "yarn workspace bridge test:fix"
+    "test:fix": "yarn workspace bridge test:fix",
+    "certora": "certoraRun --solc ~/.solc-select/artifacts/solc-0.6.11 contracts/l2/dai.sol:Dai certora/HashHelper.sol --verify Dai:certora/dai.spec --rule_sanity --solc_args \"['--optimize','--optimize-runs','200']\""
```

Comment:
This should be changed to `"certora": "yarn workspace bridge certora"`

Reply:
Done in 2a5223d.